[2024-08-04T20:52:46.646+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-04T20:52:46.716+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.carga_dimensao_campaign manual__2024-08-04T20:52:26.187913+00:00 [queued]>
[2024-08-04T20:52:46.724+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.carga_dimensao_campaign manual__2024-08-04T20:52:26.187913+00:00 [queued]>
[2024-08-04T20:52:46.724+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-04T20:52:46.747+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): carga_dimensao_campaign> on 2024-08-04 20:52:26.187913+00:00
[2024-08-04T20:52:46.758+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=3210) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-04T20:52:46.761+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'carga_dimensao_campaign', 'manual__2024-08-04T20:52:26.187913+00:00', '--job-id', '999', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpyt8dbmkn']
[2024-08-04T20:52:46.762+0000] {standard_task_runner.py:91} INFO - Job 999: Subtask carga_dimensao_campaign
[2024-08-04T20:52:46.762+0000] {standard_task_runner.py:64} INFO - Started process 3215 to run task
[2024-08-04T20:52:46.840+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.carga_dimensao_campaign manual__2024-08-04T20:52:26.187913+00:00 [running]> on host 83ccd21ce624
[2024-08-04T20:52:46.921+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Gabriel Almeida' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='carga_dimensao_campaign' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T20:52:26.187913+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-04T20:52:26.187913+00:00'
[2024-08-04T20:52:46.923+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-04T20:52:47.035+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-08-04T20:52:47.093+0000] {logging_mixin.py:188} INFO - Dados inseridos na tabela DimCampaign com sucesso.
[2024-08-04T20:52:47.094+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-04T20:52:47.094+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-04T20:52:47.111+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=carga_dimensao_campaign, run_id=manual__2024-08-04T20:52:26.187913+00:00, execution_date=20240804T205226, start_date=20240804T205246, end_date=20240804T205247
[2024-08-04T20:52:47.152+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-04T20:52:47.175+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-04T20:52:47.177+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
